package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "altis-consent-api",
+  "version": "1.0.2",
```

Comment:
If we don’t intend to publish this on npm it can be replaced with “private”: true so we can avoid needing to update it along with the plugin version.